cubeviz/image_viewer.py

```diff
@@ -218,6 +227,72 @@ def get_data_layer_artist(self, layer=None, layer_state=None):
             cls = CubevizImageLayerArtist
         return self.get_layer_artist(cls, layer=layer, layer_state=layer_state)
 
+    def _create_stats_axes(self, subset, mu, sigma):
+        rect = 0.01, 0.88, 0.15, 0.12
+        axes = self.figure.add_axes(rect, xticks=[], yticks=[])
+        circle = Circle((0.15,0.85), 0.05, color=subset.style.color)
+        axes.add_artist(circle)
+
+        text_opts = dict(x=0.05, size='smaller')
+        axes.text(**text_opts, y=0.55, s='slice: {}'.format(self._slice_index))
+        axes.text(**text_opts, y=0.30, s=r'$\mu = {:.3}$'.format(mu))
+        axes.text(**text_opts, y=0.05, s=r'$\sigma = {:.3}$'.format(sigma))
+        return axes
+
+    def _update_stats_text(self, mu, sigma):
+        mu_text = self._stats_axes.texts[1]
+        mu_text.set_text(r'$\mu = {:.3}$'.format(mu))
+        sigma_text = self._stats_axes.texts[2]
+        sigma_text.set_text(r'$\sigma = {:.3}$'.format(sigma))
+
+    def _calculate_stats(self, component, subset):
+        mask = subset.to_mask()[self._slice_index]
+        data = self._data[0][component][self._slice_index][mask]
+        return data.mean(), data.std()
+
+    def draw_stats_axes(self, component, subset):
+
+        self._component = component
+        self._subset = subset
+
+        mu, sigma = self._calculate_stats(component, subset)
+
+        if self._stats_axes is None:
```

Comment:
What if you have multiple ROIs?

Reply:
Good question: the stats are only displayed for the "active" ROI. If you have multiple ROIs, clicking on each one should update the stats to reflect that ROI. Currently the "active" ROI is only shown in the layer tree widget, but we hope to modify glue to show some kind of marker on the viewer itself indicating which ROI is active.